plugins/GSdx/GSdx.cpp

```diff
@@ -225,7 +225,7 @@ void GSdxApp::Init()
 
 	m_gs_hw_mipmapping.push_back(GSSetting(0, "Off", ""));
 	m_gs_hw_mipmapping.push_back(GSSetting(1, "Basic", "Fast"));
-	m_gs_hw_mipmapping.push_back(GSSetting(2, "Full", "Slow"));
+	m_gs_hw_mipmapping.push_back(GSSetting(2, "Full", "Very Slow - Not Recommended)"));
```

Comment:
The bracket inside the quotes needs to be removed after Recommended, it's doubled in the dialog :)